organisation/protokoll/body.md

```diff
@@ -391,7 +391,7 @@ Protokoll från samtliga Sektionsmöten (SM) och D-rektoratsmöten (DM) läggs u
 
 6/4 DM-efter-Revisions-SM ([diskussionsprotokoll](https://yoggi.datasektionen.se/protokoll/dm_efter_revisions_sm_2022)) ([beslutsprotokoll](https://yoggi.datasektionen.se/beslutsprotokoll/dm_efter_revisions_sm_2022))
 
-15/3 DM-innan-Revisions-SM ([beslutsprotokoll](https://yoggi.datasektionen.se/beslutsprotokoll/dm_innan_revisions_sm_2022))
+15/3 DM-innan-Revisions-SM ([diskussionsprotokoll](https://yoggi.datasektionen.se/protokoll/ett_dm_innan_rev_sm_dm_2022)) ([beslutsprotokoll](https://yoggi.datasektionen.se/beslutsprotokoll/dm_innan_revisions_sm_2022))
```

Comment:
Inconsistent file-naming between the two protocols (diskussions och beslut). Doesn't matter too much, but I've been trying to stick to without "ett-", at least for later protocols.

Reply:
That's true normally I would fix it, but att this point I'm so fed up with that protocol that I couldn't care less. I can tell you all about it later.